src/sentiment-analysis-api/app.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from flask import Flask, abort, jsonify, request, url_for
+import random
+import sqlalchemy as db
+
+
+app = Flask(__name__)
+
+engine = db.create_engine('postgresql+psycopg2://HIDDENALPHABET:password@hiddenalphabet-db.ceqxyonbodui.us-east-1.rds.amazonaws.com:5432/HiddenAlphabet')
+connection = engine.connect()
+metadata = db.MetaData()
+testtable = db.Table('reddit-comments', metadata, autoload=True, autoload_with=engine)
+query = db.select([testtable])
+ResultProxy = connection.execute(query)
+ResultSet = ResultProxy.fetchall()
+df = pd.DataFrame(ResultSet)
+df.columns = ResultSet[0].keys()
+def df_query(series, k):
+    
+    d = {'tsla':['tsla', 'tesla']}
```

Comment:
Can/Should this be generalized to - what appears to be - arbitrary stock tickers/company names?